### DISCUSSION TITLE
Vue support

### DISCUSSION THREAD

**Discussion by Ugzuzg** [lingui/js-lingui#1017] [Show and tell]
I am working on a Vue `Trans` component and i18n context: https://github.com/Ugzuzg/vue-lingui/tree/master/packages/lingui-vue

I was able to make the `t` macro and my `Trans` component work in Vue apps:
![image](https://user-images.githubusercontent.com/1078004/112388426-a98cb300-8ce2-11eb-8810-040b6baa339d.png)
https://github.com/Ugzuzg/vue-lingui/blob/master/examples/hello/src/components/HelloWorld.vue

Next thing on the list is extracting the messages with cli.

**Comment by semoal** [comment]
Hey that's cool!
Why not give a try to js-lingui and create a vue-extractor, I've never developed with Vue so probably i'm wrong. 
Right now we have two extractors: babel & typescript, we could build a vue one and use it for vue users?
